community/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+
+## Setting up to contribute
+
+Check out [preparing for development](PREPARING-FOR-DEVELOPMENT.md) to learn about how to setup your local dev environment with cf-for-k8s.
+
+## Pull requests
+
+If you're working on an existing issue, simply respond to the issue and express interest in working on it. This helps other people know that the issue is active, and hopefully prevents duplicated efforts.
+
+To submit a proposed change:
+
+- Fork the affected repository.
+
+- Create a new branch for your changes.
+
+- Develop the code/fix.
+
+- Add new test cases as needed. In the case of a bug fix, the tests should fail without your code changes. For new features try to cover as many variants as reasonably possible.
+
+- Modify the documentation as necessary.
+
+When ready, if you have not already done so, sign a contributor license agreements and submit the PR. The PR checks will validate your changes on a fresh cluster and run unit tests. If there are failures, check out the individual checks to debug the issue and don't hesitate to ping the members in #cf-for-k8s slack channel for help.
+
+### Getting started with contribution
+If you're looking to get started today, you can explore the [good first issue](https://github.com/cloudfoundry/cf-for-k8s/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+first+issue%22) labelled issues in cf-for-k8s repository. 
+
+If you're looking to take on more, you can expplore the [help wanted](https://github.com/cloudfoundry/cf-for-k8s/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+wanted%22) labelled issues to get started.
+
+You can create [issues](https://github.com/cloudfoundry/cf-for-k8s/issues) to report bugs or submit feature requests. The approporate templates will guide you to fill the key pieces of information.
+
+## Contributing a feature
+
+In order to contribute a feature to cf-for-k8s you'll need to go through the following steps:
+
+- First, create a feature request issue type in cf-for-k8s repository. The issue should include information about the requirements and use cases that it is trying to address. Include a discussion of the proposed design and technical details of the implementation in the issue. 
+
+- If the feature is substantial enough, A project team member will ask for architecture design doc. Create the design document in google doc and add a link to the GitHub issue. Update the projects by sending an email to cf-dev@cloudfoundry.org mailing list and/or via slack channel. Depending on the complexity and the breadth of the feature request, the project team may discuss in one of the special interest groups or during runtime PMC meetings or they might directly involve relevant projects before being approved.
```

Comment:
mailing list and/or via slack channel: Maybe it's only me, but I keep missing Slack messages and thus would prefer mailing list.